glean-core/src/upload/mod.rs

```diff
@@ -326,17 +347,7 @@ impl PingUploadManager {
         queue
     }
 
-    /// Gets the next `PingUploadTask`.
-    ///
-    /// ## Arguments
-    ///
-    /// * `glean` - The Glean object holding the database.
-    /// * `log_ping` - Whether to log the ping before returning.
-    ///
-    /// # Return value
-    ///
-    /// `PingUploadTask` - see [`PingUploadTask`](enum.PingUploadTask.html) for more information.
-    pub fn get_upload_task(&self, glean: &Glean, log_ping: bool) -> PingUploadTask {
+    fn get_upload_task_job(&self, glean: &Glean, log_ping: bool) -> PingUploadTask {
```

Comment:
```suggestion
    fn get_upload_task_internal(&self, glean: &Glean, log_ping: bool) -> PingUploadTask {
```

?

Reply:
Much nicer, clearly yesterday was not a good naming day for me.